pkg/config/data.go

```diff
@@ -529,7 +436,8 @@ func (f *File) Read(path string, in io.Reader, out io.Writer, errLog fsterr.LogI
 	if unmarshalErr != nil {
 		errLog.Add(unmarshalErr)
 
-		// The embedded config is unexpectedly invalid.
+		// If the static embedded config failed to be unmarshalled, then that's
+		// unexpected and we can't recover from that.
 		if readErr != nil {
```

Comment:
Not a blocker: while I understand the logic here, it might be better to use something else to signal that the configuration we failed to unmarshal was the static one.

Reply:
Done.